### DISCUSSION TITLE
How to determine the shell is running on a local or remote machine in theme template

### DISCUSSION THREAD

**Discussion by mrafaj** [JanDeDobbeleer/oh-my-posh#5555] [General]
I'd like to set up a theme so that it shows only my username when I'm running on my local machine but username@hostname when running on a remote machine. Is there a way to detect this so that I can configure the template accordingly?

**Comment by JanDeDobbeleer** [comment]
You can see if the `.SSHSession` variable is `true` in the `session` segment

**Reply by mrafaj** [reply]
Thanks, that works.
